src/Components/Navbar/Navbar.jsx

```diff
@@ -30,15 +38,26 @@ const Navbar = () => {
         </div>
         <ul className="navbar_links flex list_style_none">
           <li>
-            <Link to="/login" className="btn btn_primary btn_sm">
-              Login
-            </Link>
-          </li>
-          <li>
-            <Link to="/signUp" className="btn btn_outline btn_logout btn_sm">
-              Signup
-            </Link>
+            {isLoggedIn ? (
+              <button
+                to="/login"
```

Comment:
`button` element doesn't have `to` attribute. Maybe you meant `Link` here?

Reply:
Fixed in https://github.com/saadahmadhere/elmart/pull/13/commits/f03e02d76e76fbfdbad5b14631bdd33d30f3ffb7